cal/cal/models.py

```diff
@@ -551,6 +584,13 @@ def __str__(self):
             val += " for calendar {}".format(self.calendar.calendar_id)
         return val
 
+    def save(self, *args, **kwargs):
```

Comment:
This overrides the [Django save method](https://docs.djangoproject.com/en/1.10/ref/models/instances/#django.db.models.Model.save).